mantidproject/mantid
# LoadGudrunOutput improvements

**Description of work.**
* Added some UX changes:
  * Allow for multiple files to be viewed in the load dialog at a time
  * Get workspace name from loaded file meta data 

**Report to:** [Helen Playford - ISIS]

**To test:**
* Open the `LoadGudrunOutput` dialog box and click the browse button
* Locate the testing files in the build/ExternalData/Testing/Data/UnitTest directory
POLARIS00097947-min.{dcs01/mdcs01/mint01/mdor01/mgor01}
* Ensure that:
  * you can see all the valid file extensions in the browse menu.
  * you can filter what you see by changing the file extension from the drop down menu
* Select any of the workspaces and click `Run`
* Ensure that the output workspace has the same name as the top line of the input file (except `.` is replaced by `-` and no `# `)
 * Ensure that the workspace takes a custom name if you provide one. 


Fixes #23388. 

This does not require release notes because these are further changes to a new algorithm hence already covered sufficiently in the release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
